requirements/requirements.txt

```diff
@@ -13,9 +13,9 @@ asgi-amqp==1.1.3
 asgiref==1.1.2            # via asgi-amqp, channels, daphne
 asn1crypto==0.24.0        # via cryptography
 attrs==19.1.0             # via automat, service-identity, twisted
-autobahn==19.3.3          # via daphne
+autobahn==19.5.1          # via daphne
 automat==0.7.0            # via twisted
-azure-common==1.1.20      # via azure-keyvault
+azure-common==1.1.21      # via azure-keyvault
```

Comment:
It's probably fine?  But I noticed this and the pysocks requirement are getting updated here but seem unrelated to DRF?

Reply:
That's just due to [following the instructions](https://github.com/ansible/awx/blob/e95553e1b3d864c0d5056bfd719460bc9cc8a652/requirements/README.md).

If we want to avoid unrelated dependency upgrades on feature changes, then we should consider a 2-track set of instructions there, where you run `pip-compile -U` when doing general upgrades, but without the upgrade flag when adding or removing a single entry. Even doing that, I'm not sure how nicely the tool would behave. It would need testing.

Reply:
I think it comes down to what we test here prior to merge. If we're upgrading these dependencies then we need to do a regression of the Tower components that are using them. In the cases I mentioned here it would have the to Credential Management services and Notifications(?). Those are normally manual tests... otherwise we should rewrite the `requirements.[txt|in]` to not include these updates quite yet.

Reply:
They must have been pulled in by my upgrade.  I'll take a look to figure out why.